.github/workflows/manual-deploy-obscuro-gateway-database.yml

```diff
@@ -102,6 +102,7 @@ jobs:
                -v /proc/:/host/proc/:ro \
                -v /opt/datadog-agent/run:/opt/datadog-agent/run:rw \
                -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+               --log-opt max-file=3 --log-opt max-size=10m \
```

Comment:
I think it's not needed here. We have local logs only for gateway and not for gateway databse.

Reply:
Yes, but i think it's still okay to leave here. The reason is that the agent itself is emitting logs - although at a much lower rate. But if it was up a super long time, then it could potentially fill up the container fs...

```
2024-06-10 18:12:27 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:12:52 UTC | CORE | INFO | (pkg/serializer/serializer.go:454 in SendProcessesMetadata) | Sent processes metadata payload, size: 1415 bytes.
2024-06-10 18:13:27 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:14:22 UTC | PROCESS | INFO | (pkg/process/runner/runner.go:247 in logCheckDuration) | Finish container check #640 in 3.485494ms
2024-06-10 18:14:27 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:15:32 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:16:32 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:17:37 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:17:42 UTC | PROCESS | INFO | (pkg/process/runner/runner.go:247 in logCheckDuration) | Finish container check #660 in 3.250694ms
2024-06-10 18:17:52 UTC | CORE | INFO | (pkg/serializer/serializer.go:454 in SendProcessesMetadata) | Sent processes metadata payload, size: 1422 bytes.
2024-06-10 18:18:42 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:18:50 UTC | CORE | INFO | (pkg/serializer/serializer.go:429 in sendMetadata) | Sent metadata payload, size (raw/compressed): 1080/432 bytes.
2024-06-10 18:18:51 UTC | CORE | INFO | (pkg/serializer/serializer.go:429 in sendMetadata) | Sent metadata payload, size (raw/compressed): 34320/8352 bytes.
2024-06-10 18:18:51 UTC | CORE | INFO | (pkg/serializer/serializer.go:429 in sendMetadata) | Sent metadata payload, size (raw/compressed): 1978/492 bytes.
2024-06-10 18:18:52 UTC | CORE | INFO | (pkg/util/cloudproviders/cloudproviders.go:52 in DetectCloudProvider) | Cloud provider Azure detected
2024-06-10 18:18:52 UTC | CORE | INFO | (pkg/util/cloudproviders/cloudproviders.go:60 in DetectCloudProvider) | Detecting cloud provider account ID from Azure: "6e522d3f-f104-4fdf-bf88-76d2fe400cca"
2024-06-10 18:18:52 UTC | CORE | INFO | (pkg/serializer/serializer.go:429 in sendMetadata) | Sent metadata payload, size (raw/compressed): 1103/635 bytes.
2024-06-10 18:19:47 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:20:47 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
2024-06-10 18:21:02 UTC | PROCESS | INFO | (pkg/process/runner/runner.go:247 in logCheckDuration) | Finish container check #680 in 3.526283ms
2024-06-10 18:21:52 UTC | TRACE | INFO | (pkg/trace/info/stats.go:91 in LogAndResetStats) | No data received
```